modules/3d-tiles/src/ion-loader.js

```diff
@@ -0,0 +1,18 @@
+import Tiles3DLoader from './tiles-3d-loader';
+import {getIonTilesetMetadata} from './lib/ion/ion';
+
+async function preload(url, options) {
```

Comment:
`options` is a nested object for all loader modules: `image`, `fetch`, etc.

Reply:
The nested options is tricky, that means I need copy options['cesium-ion'] to/from options['3d-tiles'].